test/integration/host_test.rb

```diff
@@ -72,5 +72,11 @@ class HostIntegrationTest < ActionDispatch::IntegrationTest
       visit clone_host_path(host)
       assert page.has_no_checked_field?('host_build')
     end
+
+    test "host with .ics extension" do
```

Comment:
You put it in the `describe 'clone page'` section but this isn't the clone page. Please add it to the correct block.